scripts/translate_lessons.py

```
@@ -0,0 +1,318 @@
+
+
+def _deepl(text, lang):
+    import urllib.request
+    import urllib.parse
+
+    data = urllib.parse.urlencode({
+        "auth_key": os.environ["LLM_API_KEY"], "text": text,
+        "target_lang": lang.upper(), "tag_handling": "xml", "ignore_tags": "x",
+    }).encode()
+    req = urllib.request.Request("https://api-free.deepl.com/v2/translate", data=data)
+    with urllib.request.urlopen(req, timeout=60) as resp:
```

Ruff (0.16.0)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)